.cascade-code/Chapman.edu/_cascade/formats/Omni-Nav.vtl

```diff
@@ -0,0 +1,639 @@
+#*
```

Comment:
This is the existing Omni-Nav format.

New refactored one should behave just like this. I tested it on Dev (see Trello card comments for details) and AFAICT it does.